ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const PERMISSION_TYPES = {
+  eth_accounts: {
+    leftIcon: 'fas fa-eye',
+    label: 'eth_accounts',
+    rightIcon: null,
+  },
+};
+
+export default function PermissionsConnectPermissionList({ permissions }) {
+  const t = useI18nContext();
+  return (
+    <div className="permissions-connect-permission-list">
+      {Object.keys(permissions)
+        .map((permission) => PERMISSION_TYPES[permission])
+        .map((permission) => (
+          <div className="permission" key={permission.label}>
+            <i className={permission.leftIcon} />
+            {t(permission.label)}
```

Comment:
Using variables as the parameter to `t` does not work with the `verify-locales` strict mode, meaning that it's more difficult to detect whether the localized message is used or not. Strict mode requires that a string literal is passed in as the parameter, directly.

You could instead move `PERMISSION_TYPES` into the component, letting you use `t` directly for the `label` property:

e.g.

```
const PERMISSION_TYPES = {
  eth_accounts: {
    leftIcon: 'fas fa-eye',
    label: t('eth_accounts'),
    rightIcon: null,
  },
};
```